smithy-typescript-ssdk-libs/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "smithy-typescript-ssdk",
+  "private": true,
+  "version": "1.0.0-alpha.0",
+  "description": "Smithy TypeScript SSDK packages",
+  "main": "index.js",
+  "scripts": {
+    "clean": "yarn clear-build-cache && yarn clear-build-info && lerna clean -y",
+    "clear-build-cache": "rimraf ./*/dist ./*/types, ./*/*.tgz",
+    "clear-build-info": "rimraf **/*.tsbuildinfo",
+    "build": "lerna run build",
+    "pretest": "yarn build",
+    "test": "jest",
+    "lint": "yarn run eslint **/src/**/*.ts",
+    "format": "prettier --write **/*.{ts,js,md,json}"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/awslabs/smithy-typescript/tree/main/smithy-typescript-ssdk-libs"
+  },
+  "author": "AWS Smithy Team",
+  "license": "UNLICENSED",
+  "dependencies": {
+    "glob": "^7.1.6",
+    "rimraf": "^3.0.2"
```

Comment:
I'm surprised anything is a non-dev-dependency at this level

Reply:
That's a good point - it almost certainly is a dev dependency. I based this off of the aws client repo where it's a non-dev dependency because of some scripts they run. I don't think it really makes a difference either way though since we're not publishing this